src/PRASInterface.jl

```diff
@@ -37,6 +73,15 @@ include("ResourceAdequacy/ResourceAdequacy.jl")
 include("CapacityCredit/CapacityCredit.jl")
 end
 
+import .PRAS.assess
+import .PRAS.LOLE
+import .PRAS.EUE
+import .PRAS.val
+import .PRAS.stderror
+import .PRAS.SequentialMonteCarlo
+import .PRAS.Shortfall
```

Comment:
Do we want to import other _ResultSpecifications_ as well?

Reply:
Please check what I've added. There's a lot of boilerplate in the docs.

Reply:
> Sure, I'm fine with that. We'll export them as well, since they are the basic interface for getting results.
> 
> Downside is that some of these resultspecs are sort of bad practice: they return up to `8760*100*10000` different floating points. Users hopefully aren't actually doing that. I moved this to the bottom of the public API, but frankly, I really hope these _aren't_ commonly used by PRAS users for large samples.

Yes. We only do this if we want to export asset status for a sample based on a metric of choice. Hopefully, not many people are actually running thsi in this mode for larger systems.